Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -76,11 +76,14 @@ ProjectPamphletViewModelOutputs {
       }
       .take(first: 1)
 
-    Signal.combineLatest(freshProjectAndLiveStreamsAndRefTag, cookieRefTag)
-      .map { (project: $0.0, liveStreamEvents: $0.1, refTag: $0.2, cookieRefTag: $1) }
-      .filter { _, liveStreamEvents, _, _ in liveStreamEvents != nil }
+    Signal.combineLatest(freshProjectAndLiveStreamsAndRefTag,
+                         cookieRefTag,
+                         self.viewDidAppearAnimated.signal.ignoreValues()
+      )
+      .map { (project: $0.0, liveStreamEvents: $0.1, refTag: $0.2, cookieRefTag: $1, _: $2) }
```

Comment:
i don't think the named tuple params are necessary cause it doesnt seem like we use em. we could just do `.map { ($0.0, $0.1, $0.2, $1, $2) }`, which is just a fancy `unpack`